streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/AlertController.java

```diff
@@ -83,16 +83,15 @@ public RestResponse updateAlertConfig(@RequestBody AlertConfigParams params) {
 
   @Operation(summary = "Get alert config")
   @PostMapping("/get")
-  public RestResponse getAlertConfig(@RequestBody AlertConfigParams params) {
-    AlertConfig alertConfig = alertConfigService.getById(params.getId());
+  public RestResponse getAlertConfig(Long id) {
```

Comment:
I'm not sure if its scalability is a consideration here
I don't think it's good to change it to a parameter